app/src/atoms/buttons/OnDeviceDisplay/LargeButton.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react'
+import { css } from 'styled-components'
+import {
+  TYPOGRAPHY,
+  COLORS,
+  SPACING,
+  BORDERS,
+  NewPrimaryBtn,
+  styleProps,
+  DIRECTION_ROW,
+  Icon,
+} from '@opentrons/components'
+import { StyledText } from '../../text'
+import type { StyleProps } from '@opentrons/components'
+
+type LargeButtonTypes = 'primary' | 'secondary' | 'alert'
+interface LargeButtonProps extends StyleProps {
+  onClick: () => void
+  buttonType: LargeButtonTypes
+  buttonText: React.ReactNode
+  disabled?: boolean
```

Comment:
In the Context section, this large button component uses different icons for each button.
I think we need to pass IconName as a props.

```suggestion
  disabled?: boolean
  iconName: IconName
```

Reply:
oops good catch! I will add that, thank you!